src/rules/max-line-length/README.md

```diff
@@ -74,3 +74,37 @@ a { color: pink; background: orange; }
  */
 a { color: pink; /* this comment is also long but not on its own line */ }
 ```
+
+### `ignore: ["comments"]`
+
+Only enforce the line-length limit for lines that are not commented out.
```

Comment:
I mean "lines that do not contain comments". Or to be precise "lines that are not started with a comment".

I used the same idioms that were used to describe  `ignore: ["non-comments"]` option for this rule.